src/braket/default_simulator/openqasm/program_context.py

```diff
@@ -28,8 +15,21 @@
 )
 from ._helpers.casting import LiteralType, get_identifier_name, is_none_like
 from ._helpers.utils import singledispatchmethod
-from .braket_pragmas import parse_braket_pragma
 from .circuit import Circuit
+from .parser.braket_pragmas import parse_braket_pragma
+from .parser.openqasm_ast import (
```

Comment:
I feel absolute imports look better for consistenct; it's what we do in all our other repos too

Reply:
I actually like the relative imports when files are nearby. Especially since we have other packages that have a root name of braket, I think the relative imports cleanly show interaction between related files as opposed to a separate (yet similarly named) dependency.

Happy to chat if you still disagree

Reply:
This implementation has many relative imports, most of them not touched by this code change. We can discuss offline and if we want to enforce consistency across our packages, we'll do that as another pr